packages/roswell.rb

```diff
@@ -0,0 +1,33 @@
+require 'package'
+
+class Roswell < Package
+  description 'A lisp installer and launcher for major environment.'
+  homepage 'https://github.com/roswell/roswell'
+  version '20.01.14.104'
+  compatibility 'all'
+  source_url 'https://github.com/roswell/roswell/archive/v20.01.14.104.tar.gz'
+  source_sha256 '798b96fdcb8c89445b36692b31570bb99882d83719d6310d969ccfcb2a35a1d4'
+
+  binary_url ({
+    aarch64: '',
+     armv7l: '',
+       i686: '',
+     x86_64: '',
+  })
+  binary_sha256 ({
+    aarch64: '',
+     armv7l: '',
+       i686: '',
+     x86_64: '',
+  })
```

Comment:
You can remove this entire section.  It will be added when the pre-built binaries are generated.  No dependencies?

Reply:
Remove this entire section. Got it.
dependencies as below, Is it okay?

```
depends_on 'brotli'
depends_on 'curl'
depends_on 'krb5'
depends_on 'libcyrussasl'
depends_on 'libiconv'
depends_on 'libidn2'
depends_on 'libnghttp2'
depends_on 'libpsl'
depends_on 'libressl'
depends_on 'libssh2 '
depends_on 'libunistring'
depends_on 'openldap'
depends_on 'rtmpdump'
depends_on 'zlibpkg'
```

Reply:
You don't need to include krb5, libiconv, libidn2, libpsl, libressl, libssh2, libunistring and zlibpkg since they are already in crew "core".

Reply:
Got it. Thank you.